Dockerfile

```diff
@@ -10,19 +12,38 @@ RUN apt-get update && apt-get install -y -q eatmydata \
         zlib1g-dev \
     && rm -rf /var/lib/apt/lists/*
 
+# Install newer cmake version with globally scoped targets
+ENV CMAKE_VER=3.13.3
+RUN wget -P /opt/cmake  https://github.com/Kitware/CMake/releases/download/v${CMAKE_VER}/cmake-${CMAKE_VER}-Linux-x86_64.tar.gz \
+  ; cd /opt/cmake \
+  ; tar xzvf cmake-${CMAKE_VER}-Linux-x86_64.tar.gz \
+  ;rm -fr cmake-${CMAKE_VER}-Linux-x86_64.tar.gz
+
 RUN mkdir -p /opt/src/nifti_clib
 RUN mkdir /nifti_build
 COPY . /opt/src/nifti_clib/
 
-
 WORKDIR /nifti_build
+
+# To confirm older cmake versions can build
+# RUN cmake \
+#     -DBUILD_SHARED_LIBS=OFF \
+#     -DUSE_CIFTI_CODE=ON \
+#     -DUSE_FSL_CODE=ON \
+#     -DNIFTI_BUILD_APPLICATIONS=ON \
+#     -DTEST_INSTALL=ON \
+#     -DDOWNLOAD_TEST_DATA=OFF \
+#     /opt/src/nifti_clib \
+#     && make install \
+#     && ctest --output-on-failure -LE NEEDS_DATA
+
+ENV PATH="/opt/cmake/cmake-${CMAKE_VER}-Linux-x86_64/bin:$PATH"
 RUN cmake \
-    -DBUILD_SHARED_LIBS=ON \
-    -DNIFTI_USE_PACKAGING=ON \
+    -DBUILD_SHARED_LIBS=OFF \
```

Comment:
Shared libraries are typically more difficult to build.  Whey turn off packaging?

Reply:
Noted. Will change this.

Reply:
"If it's not tested, it's broken".  Either remove it or test it.

Reply:
Reasonable. So you don't know of anyone who uses the packaging. I know [we proposed supporting it](https://docs.google.com/document/d/1tZLNs_HEFFRxZ8Cj1_f3ddMV2tc6OmxWKMTCZK8WV5s/edit#heading=h.8v3b0htx4dtu) but I think that was me. And I would be happy with debian, an rpm distribution, macOS (likely via home-brew) and Conda for releases. After that the tar files from releases on GitHub seems adequate. In which case it can indeed be removed.